core/domain/state_domain_test.py

```diff
@@ -610,7 +610,7 @@ def test_subtitled_html_validation_with_invalid_content(self):
 
     def test_voiceover_validation(self):
         """Test validation of voiceover."""
-        audio_voiceover = state_domain.Voiceover('a.mp3', 20, True)
+        audio_voiceover = state_domain.Voiceover('a.mp3', 20, True, 24.5)
```

Comment:
Try some cases with ints too, make sure they are accepted.

Also try a case with 0.0.

Reply:
Ok will test one out.

Reply:
Will make one.

Reply:
Made a case with ints but would have to assert as invalid because the duration_sec attribute is a float.

Reply:
Unless there should be some explicit conversions involved.